board.rb

```diff
@@ -0,0 +1,324 @@
+WIDTH = 8
```

Comment:
I like the use of constants. At this point one might consider wrapping everything in a module.